docker/Dockerfile

```diff
@@ -93,7 +109,7 @@ RUN \
     openssl \
     && rm -rf /var/lib/apt/lists/*
 
-COPY --from=builder /install /usr/local
+COPY --from=builder /synapse/ /synapse
```

Comment:
/me wonders how many people rely on things being installed in `/usr/local`.

Certainly this is going to break anyone who builds docker images that install extension modules.

Reply:
#12385 has changed this to avoid the breakage.